content/en/docs/tasks/observability/metrics/tcp-metrics/index.md

```diff
@@ -107,6 +90,30 @@ will generate and collect automatically.
         Created config virtual-service/default/ratings at revision 3004
         {{< /text >}}
 
+    1.  Cleanup mongo proxy as the terminal filter:
```

Comment:
Please add a comment at the top about this being temporary. It is embedded in the config, but should be at the top as well.

Reply:
I don't think this should surface in operator-visible docs. This is fine for the developer docs, but it is extremely ugly.

Reply:
Removed this config..